### PR TITLE
Copies updates

### LINKED ISSUE #2682: Update copy on "Choosing a Native Token", Teams, Permissions, Create Expenditures screens
1) Choosing a Native Token Screen

Update copy on "Choosing a Native Token" screen while creating a colony is to remove DAI as the example and use FOX as the main example. 

"Leia completes a task for 5 FOX in the ShapeShift colony. Because FOX is the native token of the ShapeShift colony, she also earns 5 reputation in that colony."

Add in the list of examples: UNI, SUSHI, & AAVE

https://d.pr/i/tUW8Ri

--- 

2) Add "(optional)" to both 'create new team' & 'edit new team' actions (the explain why you're adding the team) https://d.pr/i/rGGfGo

---

3) Update copy on Permissions (Arbitration & Architecture) https://d.pr/i/AziVYr

Architecture -> "Create teams and manage permissions in sub-teams."

Arbitration -> (Design needs to find the copy)

---

4) Incorrect copy on second "Task" should be "Streaming" https://d.pr/buNoQb

Alternatively stated, the duplicated & 3rd item in the list should be updated from Task to Streaming.

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR updates copies around colony, details in task description

**Changes** 🏗

* Updated copies for StepTokenChoice component
* Updated copies for Edit&Create Domain dialog component
* Updated copies for Permissions checkboxes

Screenshot 2021-11-15 at 13 28 11
Screenshot 2021-11-15 at 13 26 13
Screenshot 2021-11-15 at 13 24 36
Screenshot 2021-11-15 at 13 24 24
Screenshot 2021-11-15 at 13 23 44

Resolves #2682

### CODE REVIEW

File: src/modules/dashboard/components/CreateColonyWizard/StepTokenChoice.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const MSG = defineMessages({
   },
   selectTokenSubtitle: {
     id: 'dashboard.CreateColonyWizard.StepTokenChoice.existingTokenSubtitle',
-    defaultMessage: 'For example: DAI, EOS, SNT, etc',
+    defaultMessage: 'Add in the list of examples: UNI, SUSHI, & AAVE',
```
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/CreateColonyWizard/StepTokenChoice.tsx:60]: I think Commodore meant that you should add UNI, SUSHI, and AAVE to the previous message. So the message would be something like this: `For example: DAI, EOS, SNT, UNI, SUSHI, AAVE, etc`

**Review by ArmandoGraterol — APPROVED** [review]
Looks good! Just consider what I pointed out in the comment I left

File: src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx [hunk 2]
```diff
@@ -22,11 +22,11 @@ const MSG = defineMessages({
   },
   roleDescription2: {
     id: `dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescription2`,
-    defaultMessage: 'Coming soon...',
+    defaultMessage: `Arbitration allows you to resolve disputes, make state changes, and punish bad behavior.`,
```
Comment (arrenv) [inline src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx:25]: Mostly just a curiosity, is American English preferred? e.g. behavior vs behaviour. I've noticed both used in the Dapp.

**Review by arrenv — APPROVED** [review]
Looks good. Nothing to add beyond Armando's comment.

**Pull request merged**